packages/kbn-plugin-generator/sao_template/template/public/index.ts

```diff
@@ -0,0 +1,10 @@
+export function plugin() {
+  return new <%= upperCamelCaseName %>PublicPlugin();
+}
+
+export * from '../common';
+export * from './types';
+
+// Export plugin after all other imports
+import { <%= upperCamelCaseName %>PublicPlugin } from './plugin';
+export { <%= upperCamelCaseName %>PublicPlugin as Plugin };
```

Comment:
NITs: 
- the import should be at the start of the file
- do we really need to export the `Plugin` class? isn't the factory `plugin` enough? (same goes for the `server` counterpart)